src/app/project/project-details-tab/project-details-tab.component.ts

```diff
@@ -177,11 +178,13 @@ export class ProjectDetailsTabComponent implements OnInit, AfterViewInit, OnDest
     // Disable mouse zoom on project details - iterferes with scrolling page.
     this.map.scrollWheelZoom.disable();
 
-    // Draw project-level shapefiles if they're available.
-    let shapefilesToDraw = this.shapefiles[0] || [];
+    let shapefilesToDraw = [];
     if (Array.isArray(this.project.shapefiles) && this.project.shapefiles.length > 0) {
-      shapefilesToDraw = this.project.shapefiles;
-      shapefilesToDraw = _.orderBy(shapefilesToDraw, ['order'], ['desc']);
+      // Draw project-level shape files if they're available.
+      shapefilesToDraw = _.orderBy(this.project.shapefiles, ['order'], ['desc']);
+    } else if (Array.isArray(this.shapefiles) && this.shapefiles.length > 0) {
```

Comment:
Was an array not being returned here? That would be unusual. The resolver should at least return an array unless there's some kind of error

Reply:
OK I'll take out the first part of the conditional and leave the second in.